city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,134 @@
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+import scrapy
+import datetime
+import re
+
+month_mapping = {
+    "January"  : 1,
+    "February" : 2,
+    "March"    : 3,
+    "April"    : 4,
+    "May"      : 5,
+    "June"     : 6,
+    "July"     : 7,
+    "August"   : 8,
+    "September": 9,
+    "October"  : 10,
+    "November" : 11,
+    "December" : 12,
+}
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/calendar-of-meetings"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.material_map = dict()
```

Comment:
It can be helpful to use a `defaultdict` here like in this [`cook_housing`](https://github.com/City-Bureau/city-scrapers/blob/75a4d974e4e35fd3cc349c75d1cd68f451b96b9c/city_scrapers/spiders/cook_housing.py#L22-L43) example. That way, you can already create a list of links for each date and just add links to it without having to check whether a key is already present